xlml/utils/metric.py

```diff
@@ -570,7 +570,13 @@ def get_gce_job_status(
       wait_task = current_dag.get_task(
           task_id=f"{benchmark_id}.provision.create_queued_resource.wait_for_ready_queued_resource"
       )
-    elif isinstance(task_test_config.accelerator, test_config.Gpu):
+    elif isinstance(task_test_config, test_config.PlainGpuTest):
```

Comment:
This should also catch `JSonnetGpuVmTest`

Reply:
Code updated.